index.js

```diff
@@ -23,7 +23,7 @@ var FormComponentMixin = {
   },
 
   handleChange: function handleChange(event) {
-    this.setValue(event.currentTarget.value);
+    this.setValue(event.target.value);
```

Comment:
Instead of changing the mixin itself(which would affect the handleChange for all components). Shouldn't we have this code for formsy-select only?